app/services/hyrax/collection_type_create_service.rb

```diff
@@ -0,0 +1,30 @@
+module Hyrax
+  class CollectionTypeCreateService
```

Comment:
I am glad to see this move toward a service.  The collection model is getting full.  We have some other services that are going to be coming on board too.

With that in mind, can you move the service to...  `app/services/hyrax/collection_type/create_service.rb`.  That way all the services can be gathered under that directory.

Reply:
A new push is coming soon.  The service module name had to be `CollectionTypes` to avoid namespace collision with the model, but it looks like plurality was a pattern followed for other Hyrax:: modules (i.e. Collection vs. Collections module).